docs/vehicle/commands/windows.md

```diff
@@ -9,8 +9,8 @@ Controls the windows. Will vent or close all windows simultaneously.
 | Parameter | Example | Description                                                                 |
 | :-------- | :------ | :-------------------------------------------------------------------------- |
 | command   | close   | What action to take with the windows. Allows the values `vent` and `close`. |
-| lat       | 0       | Seems to not care what this is, 0 is the only value sent by the app.        |
-| lon       | 0       | Seems to not care what this is, 0 is the only value sent by the app.        |
+| lat       | 0       | Lat and lon values must be near the car for `close` operation to succeed.   |
+| lon       | 0       | For the vent operation,  0,0 is sent from the app and works fine.           |
```

Comment:
Each description line corresponds with the row it's on, so this doesn't read right when formatted this way. I would move this text up to the description above and change each line to be about the 0 value for venting and that a nearby latitude/longitude is needed for closing.

Reply:
I think I did what you asked for here.